cmake/SFMLConfig.cmake.in

```diff
@@ -121,8 +121,8 @@ set(targets_config_file "${CMAKE_CURRENT_LIST_DIR}/SFML${config_name}Targets.cma
 if(EXISTS "${targets_config_file}")
     # Set SFML_FOUND to ON by default, may be overwritten by one of the includes below
     set(SFML_FOUND ON)
-    include("${targets_config_file}")
     include("${CMAKE_CURRENT_LIST_DIR}/SFMLConfigDependencies.cmake")
+    include("${targets_config_file}")
```

Comment:
What was the purpose of reordering these two lines?

Reply:
Wouldn't build otherwise. `SFMLConfigDependencies.cmake` depends on stuff that is introduced in `targets_config_file`.